joomla/joomla-cms
# [4.2] Update composer dependencies

### Summary of Changes
This is the result of a `composer update` to update our dependencies to their current versions as supported by our PHP minimum standard.


### Testing Instructions
1. Checkout this branch
2. Run `composer install` in the root of your site
3. Check if the frontend, backend and CLI work as expected.

### Actual result BEFORE applying this Pull Request
The site works as expected.


### Expected result AFTER applying this Pull Request
The site works as expected.


### Documentation Changes Required
None
